usr.sbin/jail/jail.conf.5

```diff
@@ -22,13 +25,12 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd September 5, 2023
+.Dd August 7, 2024
 .Dt JAIL.CONF 5
 .Os
 .Sh NAME
 .Nm jail.conf
-.Nd configuration file for
-.Xr jail 8
+.Nd configuration file for jail(8)
```

Comment:
Instead of removing useful, structured information from the manual, did you consider reporting the bug in apropos(1) to the [mandoc](https://mandoc.bsd.lv/) maintainers?

Reply:
No, I thought it wasn't a bug because mdoc(7) says to assume using child macros in a Nd is not portable.

I also, perhaps mistakenly, didn't think I was proposing removing any information, just fixing the rendering. Thanks for looking it over!

Reply:
Upon closer inspection, man.cgi produces hyperlinks for xrefs, so I will back out proposed Nd change and will file a bug instead. Thanks again.